openlibrary/templates/account/borrow.html

```diff
@@ -92,7 +92,7 @@
             $ book = get_document(loan['book'])
             $if book is None:
                 $continue
-            <tr>
+            <tr data-userid="$(loan['userid'])">
```

Comment:
This should be fine -- and I used data attributes in the Google Analytics tracking code too -- but for the record, the site is not HTML5. :)